util/network-devp2p/src/host.rs

```diff
@@ -485,7 +563,9 @@ impl Host {
 		let discovery = {
 			let info = self.info.read();
 			if info.config.discovery_enabled && info.config.non_reserved_mode == NonReservedPeerMode::Accept {
-				Some(Discovery::new(&info.keys, public_endpoint, allow_ips))
+				// enr::Enr is !Clone
+				let node_record = (*info.enr).to_base64().parse().expect("involution; qed");
```

Comment:
I don't understand what `"involution; qed"` means; proofs should be ELI5-level easy to understand!

Reply:
Reverse operation - serialize and deserialize as a surrogate copy. I will reword this.